vtu11/impl/vtu11_impl.hpp

```diff
@@ -146,6 +150,43 @@ void write( const std::string& filename,
   output.close( );
 }
 
+//ParallelWrite generates a pvtu file and accordingly the vtu pieces in a subfolder
+//Each piece consists of a set of points and cells
+template<typename MeshGenerator, typename Writer = AsciiWriter>
+void parallelWrite( const std::string& path,
+                    std::string baseName,
+                    MeshGenerator& mesh,
+                    const std::vector<DataSet>& pointData,
+                    const std::vector<DataSet>& cellData,
+                    size_t fileId, size_t numberOfFiles,
+                    Writer writer = Writer())
+{
+	//ToDo: Take care of cleaning the folder! not done in this code as Kratos takes care of it
+    fs::path p1 = path;
+    p1.make_preferred();
+    if( !fs::exists( p1 ) )
+    {
+      fs::create_directory( p1 );
```

Comment:
```suggestion
      fs::create_directories( p1 );
```

otherwise will not work for recursive paths

Reply:
what's a recursive path?

Reply:
regular path:
`restart_files_Structure`
recursive path (probably not the correct name)
`ga59vax5/WINSENT/Models_Philipp/Beam_full_model/Files_Philipp/restart_files_Structure`

I think subfolder/subpath would have been better